relay-server/src/metrics_extraction/sessions.rs

```diff
@@ -38,125 +196,163 @@ pub fn extract_session_metrics<T: SessionLike>(
         }
     };
 
-    let mut tags = BTreeMap::new();
-    tags.insert("release".to_owned(), attributes.release.clone());
-    if let Some(ref environment) = attributes.environment {
-        tags.insert("environment".to_owned(), environment.clone());
-    }
-    if let Some(client) = client {
-        tags.insert("sdk".to_owned(), client.to_owned());
-    }
+    let release = attributes.release.clone();
+    let environment = attributes.environment.clone();
+    let sdk = client.map(|s| s.to_owned());
 
     // Always capture with "init" tag for the first session update of a session. This is used
     // for adoption and as baseline for crash rates.
     if session.total_count() > 0 {
-        target.push(Metric::new_mri(
-            METRIC_NAMESPACE,
-            "session",
-            MetricUnit::None,
-            MetricValue::Counter(session.total_count() as f64),
-            timestamp,
-            with_tag(&tags, "session.status", "init"),
-        ));
+        target.push(
+            SessionsMetric::Session {
+                counter: session.total_count() as f64,
+                timestamp,
+                tags: SessionSessionTags {
+                    status: SessionStatus::Unknown("init".to_string()),
```

Comment:
I realize now that because `"init"` is not a valid session status, we should just make `SessionSessionTags::status` etc. a `String`.

Reply:
only for SessionSessionTags or for the other ones 'tags'-structs too?